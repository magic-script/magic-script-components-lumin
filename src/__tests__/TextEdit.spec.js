//
import React from "react";
import { create } from "react-test-renderer";
import { TextEdit } from "magic-script-components";

describe("TextEdit component", () => {
  test("Matches the snapshot", () => {
    const text = create(
      <TextEdit text="Test Text Edit" width={0.25} height={0.1} />
    );
    expect(text.toJSON()).toMatchSnapshot();
  });
});
