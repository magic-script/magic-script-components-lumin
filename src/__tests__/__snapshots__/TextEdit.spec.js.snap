// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextEdit component Matches the snapshot 1`] = `
<textEdit
  height={0.1}
  text="Test Text Edit"
  width={0.25}
/>
`;
